src/common/playlists/PlayList.jsx

```diff
@@ -5,6 +5,13 @@ import './playlist.css';
 
 const PlayList = () => {
   const plays = useSearchFilter();
+  if(Object.keys(plays).length===0){
```

Comment:
Isn't the `plays` an array already? Then why to do Object.keys ?

Reply:
Yes sir i understand 🤔🤔